todo_app/templates/index.html

```diff
@@ -11,7 +11,23 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        <body> 
+          <ul> 
+            {% for item in items %}
+            <li>{{ item.title }}</li>
```

Comment:
You probably want to display status as well (will be more relevant in later modules once you can update statuses).